[2024-01-19T12:16:36.523+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T12:16:35.314620+00:00 [queued]>
[2024-01-19T12:16:36.529+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T12:16:35.314620+00:00 [queued]>
[2024-01-19T12:16:36.529+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-19T12:16:36.536+0000] {taskinstance.py:2192} INFO - Executing <Task(BranchPythonOperator): CheckTableExisting> on 2024-01-19 12:16:35.314620+00:00
[2024-01-19T12:16:36.541+0000] {standard_task_runner.py:60} INFO - Started process 176 to run task
[2024-01-19T12:16:36.545+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'company_profile_pipeline', 'CheckTableExisting', 'manual__2024-01-19T12:16:35.314620+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/CrawlCompanyInfo.py', '--cfg-path', '/tmp/tmp_qf3a850']
[2024-01-19T12:16:36.546+0000] {standard_task_runner.py:88} INFO - Job 26: Subtask CheckTableExisting
[2024-01-19T12:16:36.589+0000] {task_command.py:423} INFO - Running <TaskInstance: company_profile_pipeline.CheckTableExisting manual__2024-01-19T12:16:35.314620+00:00 [running]> on host 438eff074d5a
[2024-01-19T12:16:36.672+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='anh.ho@tititada.com' AIRFLOW_CTX_DAG_OWNER='anh.ho' AIRFLOW_CTX_DAG_ID='company_profile_pipeline' AIRFLOW_CTX_TASK_ID='CheckTableExisting' AIRFLOW_CTX_EXECUTION_DATE='2024-01-19T12:16:35.314620+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-19T12:16:35.314620+00:00'
[2024-01-19T12:16:36.673+0000] {logging_mixin.py:188} INFO - Connecting to the PostgreSQL database...
[2024-01-19T12:16:36.678+0000] {logging_mixin.py:188} INFO - PostgreSQL database version:
[2024-01-19T12:16:36.679+0000] {logging_mixin.py:188} INFO - Successfully connect to Postgresql
[2024-01-19T12:16:36.680+0000] {logging_mixin.py:188} INFO - ('PostgreSQL 13.13 (Debian 13.13-1.pgdg120+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit',)
[2024-01-19T12:16:36.681+0000] {logging_mixin.py:188} INFO - Checking existing table...
[2024-01-19T12:16:36.686+0000] {python.py:201} INFO - Done. Returned value was: CreateCompanyProfileTable
[2024-01-19T12:16:36.689+0000] {branch.py:35} INFO - Branch into CreateCompanyProfileTable
[2024-01-19T12:16:36.689+0000] {skipmixin.py:178} INFO - Following branch CreateCompanyProfileTable
[2024-01-19T12:16:36.713+0000] {skipmixin.py:244} INFO - Skipping tasks [('ExistingCompanyProfile', -1)]
[2024-01-19T12:16:36.753+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=company_profile_pipeline, task_id=CheckTableExisting, execution_date=20240119T121635, start_date=20240119T121636, end_date=20240119T121636
[2024-01-19T12:16:36.804+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-19T12:16:36.827+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
